scripts/deploy_dss_sfn_queue.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import boto3
+import os
+
+sqs = boto3.resource('sqs')
+
+# Create the queue. This returns an SQS.Queue instance.
+# Amazon SQS returns error only if the request includes attributes whose values differ from those of the existing queue.
+queue = sqs.create_queue(QueueName='dss-dlq-' + os.environ["DSS_DEPLOYMENT_STAGE"], Attributes={'DelaySeconds': '5'})
+
+print(f"Queue: {queue.url}")
```

Comment:
Please replace this script with an AWS CLI invocation. It's a one liner that doesn't warrant a separate script.

Any script in scripts/ should not do anything when run with no arguments, and should print a help message when run with `--help`. (I know this is not true for some scripts in there right now - that is something we should fix)